### DISCUSSION TITLE
对于影响行数多的sql，建议加大红色字体提醒，以引起人们的注意

### DISCUSSION THREAD

**Discussion by czxin788** [hhyo/Archery#1624] [Ideas]
对于影响行数多的sql，建议加大红色字体提醒，以引起人们的注意
![image](https://user-images.githubusercontent.com/13861904/173726608-84b2142e-d38c-4075-9e8f-c1076eec6d7d.png)

**Comment by LeoQuote** [comment]
我觉得没有必要, goinception 已经加了 warning, 没必要重复
